modules/ROOT/pages/ch2-features.adoc

```diff
@@ -0,0 +1,281 @@
+|*Certificate management*|Supported|Supported|Supported
+|*Multiple Trust Stores* 
+
+*(client certs)* | Supported | Not supported |Supported
+| *Custom endpoints* | Supported | Supported | Supported
+| *Multiple custom endpoints*
+
+*per app* | Supported| Partially supported | ?
+| *TLS 1.0* | Not supported | Supported| Supported
+| *Internal endpoints* | Supported |Supported | Supported
+| *Load balancer logs* | Supported
+
+(download) | Not supported | Supported
+4+h|Network
+| *Deployment regions* | 12 geographical regions | 12 geographical regions | Supported 
+
+(manual)
+|*Virtual Private Cloud* | Fully managed
+
+(private space) | Fully managed (Anypoint VPC) | Supported 
+
+(manual)
+|*Virtual Private Network* | Fully managed
+
+(private space) | Fully managed
+
+(Anypoint VPN) | Supported 
+
+(manual)
+| *AWS Transit Gateway* | Supported
+
+(private space) | Supported | Supported
+| *Direct Connect* | Not supported | Supported
+
+(not self-serve) | Supported
+|*VPC Peering* | Not supported | Supported 
+
+(not self-serve) | Supported
+
+| *DNS* | Fully managed | Fully managed | Supported
+| *Inbound firewall rules*| Supported | Supported| Supported
+| *Outbound firewall rules*| Supported
+
+(private space) | Not supported| Supported
+|*Static IP addresses* | Supported
+
+(private space) | Supported
+
+(per app) | ?
+4+^h|App Capabilities
+|*Schedule management* | Supported | Supported | Not supported
+|*TLS certificates*
+
+*for API Gateway deployments*| Not supported | Supported | Not supported
+| *DataGraph* | Not supported | Supported | Not supported
+|*Mule patch updates* | Automated | Automated | Requires redeployment
+|*Orchestration*
+
+*management* | Fully managed | Fully managed | AWS/
+
+Self-managed
+| *Persistent object store* | Fully managed
+
+(Object Store v2)| Fully managed
+
+(Object Store v2)| Supported
+| *High availability*
+
+*across AZs* | Fully managed
+
+(with two or more replicas)|Fully managed
+
+(with two or more workers)|Supported
+
+(with two or more replicas)
+|*Secure properties* | Supported | Supported | Partially supported
+|*Log tailing*| Fully managed | Fully managed | Supported outside Anypoint Platform
+|*Continuous integration deployments*|Supported
+
+(via API and Maven plugin)| Supported
+
+(via API, Maven plugin, Anypoint CLI) |Supported 
+
+(via API and Maven plugin)
+|*Built-in notifications* | Supported | Supported |Supported
+|*Custom notifications* 
+
+*CloudHub Connector*| Supported | Supported | Not supported
+|*Log forwarding* | Manual
+
+| Supported
+
+(per app) | Manual
+|*Message broker*|Fully managed 
+
+(Anypoint MQ) | Fully managed
+
+(Anypoint MQ) | Fully managed
+
+(Anypoint MQ)
+|*App monitoring*|Fully managed
+
+(Anypoint Monitoring) | Fully managed
+
+(Anypoint Monitoring) | Fully managed
+
+(Anypoint Monitoring)
+| *Application isolation* | Supported | Supported | Supported
+|*Self-healing*| Fully managed | Fully managed | Supported
```

Comment:
What do we mean by self-healing? Auto-restart, health-check, automatic failover? Can we be more specific? @xuanshi

Reply:
auto recovery

Reply:
@vli63127 @xuanshi - should we change this to Auto-recovery?

Reply:
Changed to auto-recovery.